docs/root/configuration/other_features/dlb.rst

```diff
@@ -33,7 +33,62 @@ If enabled, the Dlb connection balancer will:
 When new connections come, one worker thread will accept it and send it to Dlb hardware. Dlb hardware
 does balancing then trigger one worker thread to receive via libevent.
 
-Installing and using Dlb
-------------------------
+Installing Dlb
```

Comment:
```suggestion
Installing DLB
```

same throughout and any existing usage of this acronym

based on https://www.intel.com/content/www/us/en/download/686372/intel-dynamic-load-balancer.html i think this should be spelt as an acronym